src/foam/nanos/http/NanoRouter.java

```diff
@@ -63,7 +63,12 @@ protected void service(final HttpServletRequest req, final HttpServletResponse r
     PM       pm         = new PM(this.getClass(), serviceKey);
 
     resp.setContentType("text/html");
-
+    resp.setHeader("X-Content-Type-Options", "nosniff");
+    resp.setHeader("Cache-Control", "no-cache, no-store, must-revalidate");
```

Comment:
Is this line correct? Don't we want some level of caching to improve performance? Currently we're downloading some small images hundreds of times for the same client session.

Reply:
it looks like the headers are only being applied to responses to post requests